packages/arb-token-bridge-ui/src/components/common/DepositCountdown.tsx

```diff
@@ -12,7 +12,7 @@ function getMinutesRemainingText(minutesRemaining: number): string {
     return 'Less than a minute...'
   }
 
-  return `~${minutesRemaining} mins remaining`
+  return `${minutesRemaining} minutes`
```

Comment:
We need to change this in E2E tests as well

Reply:
I'll fix e2e in the last PR when I'm sure there will be no further changes impacting it